kanidmd/src/lib/core/ldaps.rs

```diff
@@ -21,31 +21,33 @@ use uuid::Uuid;
 
 struct LdapSession {
     uat: Option<LdapBoundToken>,
+    eventid: Uuid,
 }
 
 impl LdapSession {
-    fn new() -> Self {
+    fn new(eventid: &uuid::Uuid) -> Self {
         LdapSession {
             // We start un-authenticated
             uat: None,
+            eventid: *eventid,
```

Comment:
It would make more sense to name this session ID I think

Reply:
Or connection_id

Reply:
I think I'm wondering if it's a good idea to have the eventid "stay the same" over multiple ldap operations because of how we do the batching of logs in the auditscope. I'm not sure - I can see how combinidng it to one is a good idea, but I can also see how it could be confusing too, because they are meant to be unique per-operation ids

Reply:
In context, connectionId would make more sense. Sessions tend to indicate some kind of long-lived thing in my head. (k)operationId's something that would be relevant for the individual operations that happen within the connection's lifetime.

(This'll be better articulated in the logging thesis I'm mostly-done writing)